akka-remote/src/main/resources/reference.conf

```diff
@@ -20,6 +20,8 @@ akka {
 
     serialization-bindings {
       "akka.actor.ActorSelectionMessage" = akka-containers
+      "akka.actor.Identify" = akka-containers
+      "akka.actor.ActorIdentity" = akka-containers
```

Comment:
We can't enable those in 2.4.x because we must be wire compatible between e.g. 2.4.4 and 2.4.5.
It's however good if they are included here as a comment, and also that we add a new section in `serialization.rst` about Akka messages that are using Java serializer and describe what should be done in application.conf to enable the new serializers.

Before merging we should also verify that all tests pass with these serializers enabled, as a one time manual test.